amq/amq62-basic.json

```diff
@@ -102,13 +102,13 @@
                     }
                 ],
                 "selector": {
-                    "deploymentConfig": "${APPLICATION_NAME}-amq"
+                    "application": "${APPLICATION_NAME}-amq"
```

Comment:
I think these should just be ${APPLICATION_NAME}, without the -amq.  If we want amq in the name, we should change the default.

Reply:
@rcernich OK, I kept with "-amq" because it was there already.
I'll update this, test and update this PR.
Thanks.

Reply:
Hmm, the "-amq" suffix needs to stay there or an additional label like "component: amq" needs to be added, otherwise there's going to be overlap between different components that are part of the same application (e.g. EAP and mysql in the eap-mysql template). 

That said, I believe I filed an issue once saying that we should use a multidimensional label system (multiple labels instead of a single label per resource), as that makes it easier for the ops team to select resources across different dimensions (e.g. all pods belonging to the same application, regardless of component; or selecting all database pods, regardless of application, etc.).

Reply:
perhaps, but we use application name for everything, which should prevent two applications from using the same name as the resources can't be instantiated.  That said, I'm assuming that we're consistent in how we apply this.  A simple test case would be to instantiate one of the eap-amq templates, then instantiate an amq template twice using "eap-app" and "eap-app-amq" for the application name and see what happens.  I suspect the deployment configs will fail to be created because they will conflict with what's already running in eap (either the eap deployment or the amq deployment).

I agree that using multiple labels for selectors would be an improvement, but maybe we should save that for 2.0.  I think that change might be a bit too much of a change in structure at this point in time.